k8s/namespaces/ia/ia-task-configuration/demo.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-task-configuration
+  annotations:
+    fluxcd.io/automated: "true"
+    fluxcd.io/tag.java: glob:pr-85-*
+    hmcts.github.com/prod-automated: disabled
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/ia/task-configuration:prod-3bd0b248
```

Comment:
I don't think we need this service deployed in demo all this does is hold the camunda artefacts